frontend/src/components/maps/leaflet/InfoSlideOut/InfoSlideOut.tsx

```diff
@@ -0,0 +1,145 @@
+import * as React from 'react';
+import { Button } from 'react-bootstrap';
+import { FaInfo } from 'react-icons/fa';
+import Control from 'react-leaflet-control';
+import styled from 'styled-components';
+import clsx from 'classnames';
+import * as L from 'leaflet';
+import { useEffect } from 'react';
+import { PropertyPopUpContext } from '../Map';
+import HeaderActions from './HeaderActions';
+import { InfoContent } from './InfoContent';
+import TooltipWrapper from 'components/common/TooltipWrapper';
+
+const InfoContainer = styled.div`
+  margin-right: -10px;
+  width: 341px;
+  min-height: 52px;
+  height: auto;
+  background-color: #fff;
+  position: relative;
+  border-radius: 4px;
+  box-shadow: -2px 1px 4px rgba(0, 0, 0, 0.2);
+  z-index: 1000;
+
+  &.closed {
+    width: 0px;
+    height: 0px;
+  }
+`;
+
+const InfoHeader = styled.div`
+  width: 100%;
+  height: 52px;
+  background-color: #1a5a96;
+  color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-evenly;
+  padding-top: 16px;
+  border-top-right-radius: 4px;
+  border-top-left-radius: 4px;
+`;
+
+const InfoMain = styled.div`
+  width: 100%;
+  height: calc(100% - 80px);
+  padding-left: 10px;
+  padding: 0px 10px 5px 10px;
+`;
+
+const InfoIcon = styled(FaInfo)`
+  font-size: 30px;
+`;
+
+const InfoButton = styled(Button)`
+  width: 52px;
+  height: 52px;
+  position: absolute;
+  top: 0;
+  left: -51px;
+  background-color: #fff;
+  color: #1a5a96;
+  border-color: #1a5a96;
+  box-shadow: -2px 1px 4px rgba(0, 0, 0, 0.2);
+  &.open {
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+    top: 0px;
+  }
+`;
+
+const Title = styled.p`
+  font-size: 18px;
+  color: #ffffff;
+  text-decoration: none solid rgb(255, 255, 255);
+  line-height: 18px;
+  font-weight: bold;
+`;
+
+export type IInfoControl = {
+  /** whether the slide out is open or closed */
+  open: boolean;
+  /** set the slide out as open or closed */
+  setOpen: () => void;
+  /** additional action for when a link is clicked */
+  onHeaderActionClick?: () => void;
+};
+
+/**
+ * Component to display the popup information of a parcel/building
+ * @param open open/closed state of the slideout
+ * @param setOpen function to set the slideout as open or closed
+ * @param onHeaderActionClick action to be taken when a menu item is clicked
+ */
+const InfoControl: React.FC<IInfoControl> = ({ open, setOpen, onHeaderActionClick }) => {
+  const popUpContext = React.useContext(PropertyPopUpContext);
+
+  useEffect(() => {
+    const elem = L.DomUtil.get('infoContainer');
+    if (elem) {
+      L.DomEvent.on(elem!, 'mousewheel', L.DomEvent.stopPropagation);
```

Comment:
you probably want to turn this event listener off when this component is cleaned up. useeffect returns a function that is invoked when the component is destroyed, so you can put the L.DomEvent.off(...) there.

Reply:
I just copied this over from the layers component, I'm not actually sure what this is doing, what do you mean by "cleaned up"?

Reply:
What feature is this for?

Reply:
The slideout itself. Can I just remove this useEffect, @devinleighsmith ? It doesn't seem to effect functionality if I do

Reply:
If this is reused code I'll just log some tech debt for this.